docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md

```diff
@@ -0,0 +1,37 @@
+# What is Nebula Operator
+
+## Concept of Nebula Operator
+
+Nebula Operator is a tool to automate the deployment, and operation and maintenance of [Nebula Graph](https://github.com/vesoft-inc/nebula) clusters on [Kubernetes](https://kubernetes.io). Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real [cloud-native graph database](https://www.nebula-cloud.io/).
+
+## How it works
+
+For resource types that do not exist within Kubernetes，you can register them by adding custom API objects. The common way is to use the [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions).
+
+Nebula Operator abstracts the deployment management of Nebula Graph clusters as a CRD. By combining multiple built-in API objects including StatefulSet, Service, and ConfigMap, the routine management and maintenance of a Nebula Graph cluster are coded as a control loop in the Kubernetes system. When a CR instance is submitted, Nebula Operator drives database clusters to the final state according to the control process.
```

Comment:
```suggestion
Nebula Operator abstracts the deployment management of Nebula Graph clusters as a CRD. By combining multiple built-in API objects including StatefulSet, Service, and ConfigMap, the routine management, and maintenance of a Nebula Graph cluster are coded as a control loop in the Kubernetes system. When a CR instance is submitted, Nebula Operator drives database clusters to the final state according to the control process.
```

Reply:
the subject is maintenance and management, will keep the original content.